server/compile_lexicon_to_json.py

```diff
@@ -166,70 +198,73 @@ def compile_to_json_full_cognates(path, proto, cognates="cogid"):
     # create the board dictionary
     doculects = sorted(set([row["DOCULECT"] for row in data_dict.values()]))
     boards = {
-            "fstDoculects": doculects,
-            "fstUp": {d: {} for d in doculects},
-            "fstDown": {d: {} for d in doculects},
-            "boards": {},
-            "words": {},
-            "columns": {},
-            "syllables": {},
-            "boards": {}
-            }
+        "fstDoculects": doculects,
+        "fstUp": {d: {} for d in doculects},
+        "fstDown": {d: {} for d in doculects},
+        "boards": {},
+        "words": {},
+        "columns": {},
+        "syllables": {},
+    }
 
     # fill data with content by iterating over the data_dict
     for i, row in data_dict.items():
-        idx = "word-"+str(i)
+        idx = "word-" + str(i)
         boards["words"][idx] = {
-                "id": idx,
-                "doculect": row["DOCULECT"],
-                "syllables": [".".join(row["TOKENS"].split())],
-                "gloss": row["CONCEPT"],
-                "glossid": row["CONCEPT"]
-                }
-        boards["syllables"][idx+"-0"] = {
-                "id": idx+"-0",
-                "doculect": row["DOCULECT"],
-                "syllables": boards["words"][idx]["syllables"],
-                "gloss": boards["words"][idx]["gloss"],
-                "glossid": boards["words"][idx]["glossid"],
-                "wordID": idx,
-                "syllOrder": 0,
-                "syllable": boards["words"][idx]["syllables"][0]
-                }
+            "id": idx,
+            "doculect": row["DOCULECT"],
+            "syllables": [".".join(row["TOKENS"].split())],
+            "gloss": row["CONCEPT"],
+            "glossid": row["GLOSSID"],
```

Comment:
GLOSSID is not supported in all datasets, and may confuse users as input. I would even say it is not essential for CAPR, so I used "CONCEPT" to make sure the boards.json has the same structure.